tasks/certificate.yml

```diff
@@ -10,7 +10,10 @@
     command: >
       ./easyrsa
       --batch
+      --dn-mode=org
```

Comment:
Why *always* set the mode to org, even if those parameters will now default to the EasyRSA default settings if you don't specify them manually?

Reply:
I can solve this by defaulting all values to the empty string. Assuming the CN is parsed by TLS libs correctly this should not harm and simplifies the logic of the role

Reply:
Hmm...why not do both? You can use something like `{{ '--dn-mode' if whatever else None }}` _and_ default them in the EasyRSA config. That way, existing certificates stay the same and new certs don't get strange defaults if you only set one of `O`/`OU`.

Reply:
The interesting `if whatever` will check all 6 DN fields because if either is defined we have to use the extended form. Please confirm that you really want this.